src/Core/src/Platform/Windows/NavigationViewExtensions.cs

```diff
@@ -37,13 +37,22 @@ public static void UpdateTopNavigationViewItemTextColor(this MauiNavigationView
 					navigationView.TopNavArea.Resources.Remove("TopNavigationViewItemForegroundPointerOver");
 					navigationView.TopNavArea.Resources.Remove("TopNavigationViewItemForegroundPressed");
 					navigationView.TopNavArea.Resources.Remove("TopNavigationViewItemForegroundDisabled");
+
+					navigationView.TopNavArea.Resources.Remove("NavigationViewItemForeground");
+					navigationView.TopNavArea.Resources.Remove("NavigationViewItemForegroundPointerOver");
+					navigationView.TopNavArea.Resources.Remove("NavigationViewItemForegroundPressed");
```

Comment:
And what about `NavigationViewItemForegroundPressed`?

Reply:
@jsuarezruiz, It seems that the [NavigationViewItemForegroundDisabled](https://github.com/microsoft/microsoft-ui-xaml/blob/268e06867114c04cc0f25201b57f64e326113ba0/src/controls/dev/NavigationView/NavigationView_themeresources.xaml#L24) color is not used for the Chevron icon in the NavigationView. Could you please check and let me know if you have any concerns?